frontend/src/scenes/insights/utils/cleanFilters.ts

```diff
@@ -245,9 +245,15 @@ export function cleanFilters(
             cleanSearchParams.actions = []
         }
 
+        if (
+            (cleanSearchParams.date_from === cleanSearchParams.date_to || cleanSearchParams.date_from === 'dStart') &&
+            cleanSearchParams.interval !== 'hour'
+        ) {
+            // If viewing a single data point (yesterday or today), change to bar chart as the line chart will look empty
+            cleanSearchParams.display = ChartDisplayType.ActionsBarChart
+        }
+
         return cleanSearchParams
-    } else if ((filters.insight as string) === 'HISTORY') {
-        return cleanFilters({ insight: InsightType.TRENDS })
```

Comment:
This is an unrelated change... It might be good to keep this "legacy" check around as well... Maybe 🤷